docs/administration/recent_activity/recent_activity.md

```diff
@@ -51,6 +51,17 @@ This policy is required to view [activity log in user profile]([[= user_doc =]]/
 
     Do not assign `activity_log/read` permission to the Anonymous role, even with the owner limitation, because this role is shared among all unauthenticated users.
 
+## User privacy
+
+!!! caution
+
+    The username of the User performing the action is logged. When acting through HTTP, User's IP is also logged.
```

Comment:
This should probably clarify what is meant by HTTP here. HTTPS does not shield IP addresses, so is this about using a frontend through a browser directly, vs. using an app through the API? Won't the API log IPs?

Or is it only console commands that won't log IP, like in the migration example? That would make sense.

Reply:
@glye 
There is an IP as soon as you go through the web server. There is no IP when you act from server host shell (such as what you cited, migration, or any console command).

Event if you are connected to the host server through SSH over Internet, and got an IP associated to your shell session, this IP is not logged.

Reply:
Right, as I hoped. Could you clarify this, please? Maybe:
```suggestion
    The username of the User performing the action is logged. When acting through the web server, the User's IP address is also logged. Other access, such as console commands, does not log an IP.
```

Reply:
Just https://learn.microsoft.com/en-us/style-guide/word-choice/use-contractions
```suggestion
    The username of the User performing the action is logged. When acting through the web server, the User's IP address is also logged. Other access, such as console commands, doesn't log an IP.
```